qutip/pim/dicke.py

```diff
@@ -209,13 +209,13 @@ def get_j_m(N, index):
 
     For N = 6 |j,m> would be :
 
-    | 3, 3>
-    | 2, 3>  | 2, 2>
-    | 1, 3>  | 1, 2>  | 1,  1>
-    | 0, 3>  | 0, 2>  | 0,  1> |0, 0>
-    |-1, 3>  |-1, 2>  |-1,  1>
-    |-2, 3>  |-2, 2>
-    |-3, 3>
+    | 3, 3  >
```

Comment:
@nathanshammah Is this definition and values for |j, m> correct ? And the test ? This tests all the elements which are valid dicke state elements and gets the |j, m> values for them.